pdudaemon/pdurunner.py

```diff
@@ -19,14 +19,17 @@
 #  MA 02110-1301, USA.
 
 import asyncio
+import concurrent.futures
 import logging
 import time
 import traceback
+
 import pexpect
-import concurrent.futures
+
+import pdudaemon.drivers.strategies
 from pdudaemon.drivers.driver import PDUDriver
-import pdudaemon.drivers.strategies  # pylint: disable=W0611
 
+assert pdudaemon.drivers.strategies, "This is a required unused import"
```

Comment:
not sure why this unused import broke the tests, but used this dummy assert to highlight that it is needed, happy to take alternative suggestions for the actual comment

Reply:
This is used to load all drivers, which is necessary as PDUDriver iterates over it's subclasses to find drivers at runtime.

Reply:
thanks, assert statement updated